ui/decredmaterial/editor.go

```diff
@@ -69,28 +84,31 @@ func (t *Theme) Editor(editor *widget.Editor, hint string) Editor {
 		m2: unit.Dp(2),
 		m5: unit.Dp(5),
 
-		pasteBtnMaterial: IconButton{
+		showHidePassword: new(widget.Clickable),
+		iconhide:         &widget.Image{Src: paint.NewImageOp(t.decredIcons["ic_hide_password"])},
+
+		clearBtMaterial: IconButton{
 			material.IconButtonStyle{
-				Icon:       mustIcon(widget.NewIcon(icons.ContentContentPaste)),
+				Icon:       mustIcon(widget.NewIcon(icons.ContentClear)),
 				Size:       m25,
 				Background: color.NRGBA{},
 				Color:      t.Color.Text,
 				Inset:      layout.UniformInset(m0),
 				Button:     new(widget.Clickable),
 			},
 		},
-
-		clearBtMaterial: IconButton{
+		showPasswordButton: IconButton{
 			material.IconButtonStyle{
-				Icon:       mustIcon(widget.NewIcon(icons.ContentClear)),
+				Icon:       mustIcon(widget.NewIcon(icons.ImageRemoveRedEye)),
 				Size:       m25,
```

Comment:
The size of the image should be 24 not 25. Kindly log into figma and have a look at the mockups for precise dimensions.